spec/requests/option_set_spec.rb

```diff
@@ -183,6 +183,11 @@
     end
   end
 
+  it "exports successfully" do
```

Comment:
Thanks for adding this spec too

Reply:
It is nice to be thorough but I don't think we actually need this as it simply duplicates the feature spec unless I'm missing something...

I think our goal is to use feature specs where possible and request specs only where there is no browser-accessible UI to exercise. So unless I'm off base here I think we should remove this spec and just leave the feature spec for the sake of simplicity.